modules/installation-guide/partials/proc_importing-untrusted-tls-certificates.adoc

```diff
@@ -12,7 +14,8 @@ Typical cases that may require this addition are:
 * when the underlying Kubernetes cluster uses TLS certificates signed by a CA that is not trusted,
 * when {prod-short} server or workspace components connect to external services such as {identity-provider} or a Git server that use TLS certificates signed by an untrusted CA.
 
-To store those certificates, {prod-short} uses a dedicated ConfigMap. Its default name is `ca-certs` but {prod-short} allows configuring its name.
+{prod-short} uses labeled config maps in {prod-short} namespace as sources for TLS certificates.
+The config maps can have arbitrary number of data fields with arbitrary number of certificates each.
```

Comment:
`arbitrary number of data fields` -> `arbitrary number of keys`

Reply:
If we are talking about a config map `data field` is understandable for sure. If we talk about keys, it would require more context from a reader, imo.